gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/NetworkService.java

```diff
@@ -34,15 +34,14 @@
 import java.nio.file.Paths;
 import java.time.OffsetDateTime;
 import java.time.format.DateTimeFormatter;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Properties;
+import java.util.*;
```

Comment:
.* is not ideal

Reply:
Import .* is still not ideal. Here, by not specifying explicitly which classes you want to import from java.util package, you are importing about 100 classes contained in this package!

Reply:
Ok